quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus/src/main/java/com/redhat/microprofile/jdt/internal/quarkus/QuarkusPlugin.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.internal.quarkus;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+/**
+ * The activator class controls the Quarkus JDT LS Extension plug-in life cycle
+ */
+public class QuarkusPlugin implements BundleActivator {
```

Comment:
Do we really need this activator ? I've seen core platform devs are in the process of removing as much activators as they can

Reply:
I didn't know that, thanks @jeffmaury for this info. Do you know the reason about that? (I suppose that it's for performance)

I think we can remove it because I'm using just the PLUGIN_ID String constants.

Reply:
@jeffmaury I removed the QuarkusPlugin.

Reply:
Yes performance is the root cause